tests/unit/conftest.py

```diff
@@ -1130,3 +1131,13 @@ def g_tester_id(app, db):
     )
     db.session.commit()
     return tester.id
+
+
+@pytest.fixture
+def iiif_cache():
```

Comment:
I think it's better to pass the `app` fixture
```suggestion
def iiif_cache(app):
```

And then:

```python
cache = app.extensions['iiif'].cache
```

so that you use the application config.